test/main_test.go

```diff
@@ -399,3 +313,643 @@ func (suite *JsonHttpApiSuite) TestMiddlewares_NoPayload() {
   assert.Equal(suite.T(), http.StatusUnauthorized, recorder.Code)
   assert.Equal(suite.T(), "", recorder.Body.String())
 }
+
+func (suite *JsonHttpApiSuite) TestCORS_WithoutAnyHandler() {
+  request, _ := http.NewRequest(http.MethodOptions, "/resources", nil)
+  recorder := httptest.NewRecorder()
+
+  suite.handler.ServeHTTP(recorder, request)
+
+  assert.Equal(
```

Comment:
NIT, `assert.Empty` should achieve the same and save couple characters

Reply:
Much better. Thanks.

Reply:
Done in https://github.com/renra/go-json-http-handler/pull/10/commits/2d26dcf8a077056406c2dc2d091053bf6cefce5e